tests/libsubid/04_nss/test_range

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -x
```

Comment:
> If stderr has a _ in the output (as in dirname libsubid/04_nss),
> then autopkgtest, which is used in debian CI, calls that a failure.

Out of curiosity, could you point to documentation for that?  From what I can see, it considers any stderr output at all a failure (except if `allow-stderr`):
<https://salsa.debian.org/ci-team/autopkgtest/-/blob/master/doc/README.package-tests.rst>.
And didn't find anything special about underscores.

Reply:
Nope, can't point to documentation, that was just my experience :)

Reply:
I'm quite sure removing -x will not be necessary.

Reply:
I'll drop this patch (set -x).  Please open a separate PR for fixing that.